homeassistant/components/homewizard/sensor.py

```diff
@@ -41,6 +49,11 @@
         icon="mdi:wifi",
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    SensorEntityDescription(
+        key="active_tariff",
+        name="Active tariff",
+        icon="mdi:calendar-clock",
```

Comment:
As this can only have 2 options, we can mark it as an enum. This will help the automation & scripts UI to show possible options when automating on these values.

```suggestion
        icon="mdi:calendar-clock",
        device_class=SensorDeviceClass.ENUM,
        options=["1", "2"],
```

Reply:
Added this and tariff 3 + 4 for the few who have that in https://github.com/home-assistant/core/pull/85198/commits/34a6cee8212cc89629a22c4e821800eeb8c9fb6a